hack/make/test-integration-flaky

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+source hack/validate/.validate
+new_tests=$(
```

Comment:
Does this need to be exported to be picked up in the sub shell?

If so might want to run all of it in a subshell to not lean the var.

Reply:
No it does not need it. I'm not exactly sure why but it looks working fine on current logic (I just double checked with my example flaky test)